zerver/migrations/0001_initial.py

```diff
@@ -47,8 +53,8 @@ class Migration(migrations.Migration):
 CREATE TRIGGER zerver_message_update_search_tsvector_async
   BEFORE INSERT OR UPDATE OF subject, rendered_content ON zerver_message
   FOR EACH ROW EXECUTE PROCEDURE append_to_fts_update_log();
-"""),
-            ]
+""")
+        zulip_postgres_migrations = [full_text_search_migration]
 
```

Comment:
This approach of modifying the initial migration to use PGROONGA isn't ideal, since it makes it fairly unpleasant for an existing Zulip installation to switch to using PGROONGA.  I'm not sure what the right answer is (maybe we just provide a separate script the user can use when they toggle PGROONGA it to create the migration), but wanted to flag this as a potential issue.